src/components/Drawer/Drawer.js

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import { Box, Drawer, Toolbar, Divider, IconButton, List, ListItem, ListItemText, ListItemIcon, Avatar, Typography } from "@material-ui/core";
+import MenuIcon from '@mui/icons-material/Menu';
+import EmailIcon from '@mui/icons-material/Email';
+import FormatListNumberedIcon from '@mui/icons-material/FormatListNumbered';
+import PeopleIcon from '@mui/icons-material/People';
+import LogoutIcon from '@mui/icons-material/Logout';
+import CloseIcon from '@mui/icons-material/Close';
+
+const drawerWidth = 250;
+
+const DrawerHeader = styled('div')(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  padding: theme.spacing(0, 1),
+  // necessary for content to be below app bar
+  ...theme.mixins.toolbar,
+  justifyContent: 'flex-start',
+}));
+
+export default function PersistentDrawerRight() {
+  const [open, setOpen] = React.useState(false);
+
+  const handleDrawerOpen = () => {
+    setOpen(true);
+  };
+
+  const handleDrawerClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <Box sx={{ display: 'flex' }}>
+        <Toolbar>
+          <IconButton
+            color="inherit"
+            aria-label="open drawer"
+            edge="end"
+            onClick={handleDrawerOpen}
+            sx={{ ...(open && { display: 'none' }) }}
+          >
+            <MenuIcon />
+          </IconButton>
+        </Toolbar>
+      <Drawer
+        sx={{
+          width: drawerWidth,
+          flexShrink: 0,
+          '& .MuiDrawer-paper': {
+            width: drawerWidth,
+          },
+        }}
+        variant="persistent"
+        anchor="right"
+        open={open}
+      >
+        <DrawerHeader>
+          <IconButton onClick={handleDrawerClose}>
+            <CloseIcon />
+          </IconButton>
+        </DrawerHeader>
+        <List>
+          <Box display="flex" flexDirection="column" alignItems="center" justifyContent="center" margin={2}>
+            <Avatar alt="Zdjęcie" src="" />
+            <Typography>Marcin</Typography>
```

Comment:
Tutaj powinno pobierać to ze stora system.

Reply:
chyba ok